imports/client/markdown.ts

```diff
@@ -1,17 +1,6 @@
 import DOMPurify from 'dompurify';
-import marked from 'marked';
-
-(marked.InlineLexer.rules.gfm as marked.Rules).em = /^\b_((?:__|[^_])+?)_\b/;
-(marked.InlineLexer.rules.gfm as marked.Rules).strong = /^\*\b((?:\*\*|[^*])+?)\b\*/;
-
-const renderer = new class extends marked.Renderer {
-  link(href: string, title: string, link: string) {
-    const realLinkFunc = marked.Renderer.prototype.link.bind(this);
-    return realLinkFunc(href, title, link)
```

Comment:
I think we probably do still want to make links open in a new tab (`target="_blank"`), with `noopener`/`noreferrer`?

Reply:
Oh yeah, good call. Fixed